### ISSUE TITLE
question about value of base learning rate

### ISSUE THREAD

**Issue by liyunsheng13** [PeizeSun/SparseR-CNN#54]
Hi Peize
When I read the yaml file that contains learning parameters, I find the base learning rate is just 2.5e-5, which is much smaller than the learning rate (0.02) used in detectron2. I wonder why the learning rate is so small. Have you tried to train the network with larger learning rate?

Thanks

**Comment by iFighting** [comment]
> Hi Peize
> When I read the yaml file that contains learning parameters, I find the base learning rate is just 2.5e-5, which is much smaller than the learning rate (0.02) used in detectron2. I wonder why the learning rate is so small. Have you tried to train the network with larger learning rate?
> 
> Thanks

we use adam optimizer, so the base learning rate is just 2.5e-5, if we use larger lr, the network will not converge

**Comment by iFighting** [comment]
> Hi Peize
> When I read the yaml file that contains learning parameters, I find the base learning rate is just 2.5e-5, which is much smaller than the learning rate (0.02) used in detectron2. I wonder why the learning rate is so small. Have you tried to train the network with larger learning rate?
> 
> Thanks

if you want to have more discussion with us, you just can add my wechat: Eureka_Yi
